website/versioned_docs/version-0.60/text.md

```diff
@@ -14,32 +14,25 @@ In the following example, the nested title and body text will inherit the `fontF
 import React, { Component } from 'react';
```

Comment:
The file correct for changes is 'docs/text.md'

Reply:
yes in the last commit i change this.

Reply:
https://github.com/facebook/react-native-website/pull/1617/files